Practice4/Validator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace Practice4
+{
+    public static class Validator
+    {
+        public const int MAX_ATTEMPTS = 10;
+
+        public static char[] GetValidatedCharsArray()
+        {
+            Console.WriteLine("\nPlease populate a word or a phrase to check");
+            int attempt = 0;
+            while (attempt < MAX_ATTEMPTS)
+            {
+                string inputString = Console.ReadLine();
+                if (inputString.ToString() != string.Empty)
+                {
+                    try
+                    {
+                        char [] populatedCharactersArray= inputString.ToCharArray();
+                        bool numberFlag = false;
+                        foreach (char character in populatedCharactersArray)
+                        {
+                            if (Char.IsDigit(character)||(!Char.IsLetterOrDigit(character)))
```

Comment:
please, simplify the name

![image](https://user-images.githubusercontent.com/9382934/118272403-3c6bf180-b4cb-11eb-8895-1504d25c8d5f.png)